doc/source/Linux.rst

```diff
@@ -0,0 +1,194 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration of several of the plugins available from
+
+Acquiring memory
+----------------
+
+Volatility3 does not provide the ability to acquire memory. In this tutorial we will see how we can use  `LiME <https://github.com/504ensicslabs/lime>`_ for this purpose. 
+It supports 32 and 64 bit captures from native Intel hardware as well as virtual machine guests. 
+It also supports capture from Android devices. See below for example commands building and running LiME:
+
+.. code-block:: shell-session
```

Comment:
Will these instructions get out of date?  I'm wary about including specific implementation instructions which may get old/stale, given how often documentation gets reviewed/refreshed.

Reply:
I'm not sure if LIME will be working on future architecture. So Should I remove the line specifying the architecture?

Reply:
It's not really whether they work with a future architecture, it's more whether they decide to move to cmake or meson or some other build system that makes these instructions incorrect.  I know it's not as "one stop" as providing direct instructions for compilation in our documentation, but I'd much prefer directing people at the packages documentation for figuring out how to compile it, than including the information ourselves.  Either that, or state that these commands are representative of compilation, but you should always read the package's documentation to ensure you're using the latest instructions.